src/main/views/subscription-management.njk

```diff
@@ -1,10 +1,115 @@
+        items: [{
+            text: 'All subscriptions (' + allSubscriptions + ')',
+            href: '?all',
+            active: activeAllTab
+        }, {
+            text: 'Subscriptions by case (' + caseTableData | length + ')',
+            href: '?case',
+            active: activeCaseTab
+        }, {
+            text: 'Subscriptions by court or tribunal (' + courtTableData | length + ')',
+            href: '?court',
+            active: activeCourtTab
+        }]
+    }) }}
+    {% if caseTableData | length == 0 and courtTableData | length == 0 %}
+        <p class="govuk-body">You currently have no subscriptions</p>
+    {% else %}
+        {% if activeAllTab == true or activeCaseTab == true %}
+            {# Case Subscriptions Table #}
+            {{ govukTable({
+                attributes: {
+                    'data-module': 'moj-sortable-table',
+                    'id': 'cases-table'
+                },
+                classes: 'govuk-!-margin-bottom-9',
+                head: [
+                    {
+                        text: "Case name",
+                        attributes: {
+                        "aria-sort": "ascending"
+                    },
+                        classes: 'govuk-!-width-one-third'
+                    },
+                    {
+                        text: "Case reference number",
+                        attributes: {
+                        "aria-sort": "none"
+                    },
+                        classes: 'govuk-!-width-one-third'
+                    },
+                    {
+                        text: "Date added"
+                    },
+                    {
+                        text: "Actions",
+                        format: "numeric"
+                    }
+                ],
+                rows: caseTableData
+            }) }}
+        {% endif %}
+        {% if activeAllTab == true or activeCourtTab == true %}
+            {# Court Subscriptions Table #}
+            {{ govukTable({
+                attributes: {
+                    'data-module': 'moj-sortable-table',
+                    'id': 'courts-table'
+                },
+                head: [
+                    {
+                        text: "Court or tribunal name",
+                        attributes: {
+                        "aria-sort": "ascending"
+                    },
+                        classes: "govuk-!-width-two-thirds"
+                    },
+                    {
+                        text: "Date added"
+                    },
+                    {
+                        text: "Actions",
+                        format: "numeric"
+                    }
+                ],
+                rows: courtTableData
+            }) }}
+        {% endif %}
+    {% endif %}
+{% endblock %}
+
+{% block bodyEnd %}
+    <script type="text/javascript" src="/assets/js/jquery.js"></script>
+    <script type="text/javascript" src="/assets/js/mojAll.js"></script>
```

Comment:
Is the mojAll needed here? I'm thinking it might already be included in the moj macro's

Reply:
If I exclude these 2 files column sorting doesn't work